package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "spring",
+    "version": "1.0.0",
+    "description": "",
+    "main": ".prettierrc.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "lint": "eslint .",
+        "lint --fix": "eslint . --fix"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "devDependencies": {
+        "@eslint/js": "^9.10.0",
+        "eslint": "^9.10.0",
+        "eslint-config-prettier": "^9.1.0",
+        "eslint-plugin-prettier": "^5.2.1",
```

Comment:
Eslint has abandoned formatting rules so we  don't need that anymore
https://eslint.org/blog/2023/10/deprecating-formatting-rules/